core/models.py

```diff
@@ -5,7 +5,7 @@
 from django.core.cache import cache
 from django.core.exceptions import ValidationError
 from django.db import models
-from django.template.defaultfilters import slugify
+from django.utils.text import slugify
```

Comment:
confusingly, Django now has 2 `slugify()` functions, but only the utils.text one supports `allow_unicode` param.